CMakeLists.txt

```diff
@@ -168,6 +173,18 @@ macro(lapack_install_library lib)
   )
 endmacro()
 
+else()
+
+macro(lapack_install_library lib)
```

Comment:
In my opinion it should suffice to use a function here.

Reply:
Is it advisable to use functions instead of macros? Macro and function will work the same in this case, right?

Reply:
I'm out of my league here. I am rather strictly cargo-coding CMake and hope that what works, is also somewhat correct.